src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -253,15 +255,16 @@ public String getColumnValue(final VariantContext variant, final boolean forceLo
                     return "";
                 }
 
-                if (variant.getAlleles().size() == 3) { // GT 0/1 1/1
-                    out = "|" + outNotAlleleSpecific + ",1|";
-
-                } else if (variant.getAlleles().size() == 4) { // GT 1/2
-                    // TODO: just replicate rather than distribute, is this right?
-                    out = "|" + outNotAlleleSpecific + ",1|" + outNotAlleleSpecific + ",1|";
-                } else {
-                    throw new UserException("Expected diploid sample to either have 3 alleles (ref, alt, non-ref) or 4 alleles (ref, alt 1, alt 2, non-ref)");
+                // indexing at 0 earlier guarantees that this should not have an indexing error
+                int numVariants = variant.getAlleles().size() - 2;
+                // Moving this TODO note up here as an open question for how we want to handle this quantity
+                // with more than one allele
+                // TODO: just replicate rather than distribute, is this right?
+                out = "|";
+                for (int i = 0; i < numVariants; ++i) {
+                    out += outNotAlleleSpecific + ",1|";
                 }
+                // As of VS-910, we aren't going to consider any number of alleles to necessarily be a UserException
```

Comment:
so I did consider it and even started doing so.  But as I started pulling this out and began parameterizing it, it looked like it would be more work than it's worth.  Yes, we see this repeated in two places.  But would it be repeated exactly like this in a third?  Would they have the string ",|" added to them every time?  The argument list started getting large enough that it seemed like a lot of work for something that might not be repeated a third time again.  If/when we redo and expend this code for long reads (or just in a cleanup pass with methods) and we find ourselves doing something similar again, I think that'll be a good time to see what a general version really looks like.